releases/net472/KnownIssues/613745 - Single-name references are removed by the SDK when targeting 4.7.2.md

```diff
@@ -0,0 +1,23 @@
+# Single-name references are removed by the SDK when targeting 4.7.2 and referencing a package with the same assembly.
+
+## Symptoms
+When a project targets 4.7.2 and references an assembly with both a single name reference and a package reference (either a explicit or a transitive package reference), the assembly will not be passed as a reference to the compiler, causing build time errors for types that are not found. An example of a build error caused by this issue is:
```

Comment:
a explicit --> an explicit
will not be passed --> is not passed
build time --> build-time